### DISCUSSION TITLE
File management

### DISCUSSION THREAD

**Discussion by Dimonka01** [whyboris/Video-Hub-App#626] [Ideas]
Hi Boris,
This is a really great tool!
What I am really missing in it is to have possibility move files between folders within a hub and maybe to have possibility to move/copy files to another hub. Sort of file management features. For this I guess it will be good to have some select and multiselect features in the main view.. And make video files open by double click.

And when I am thinking further it would be nice to define a automatic rule what is expected to be in a folder to move into the folder by a rule. Like setup a rule for folder "2021" to move all files that contains 2021 in the name into this folder. Ideally with some confirmation.

I hope my ideas are making sense.
Best

**Comment by whyboris** [comment]
Thanks for creating this topic! It's related to #484 👍 
You created #627 as well - we can continue the discussion there or here, I'm happy either way 🤝
